src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +29,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.debug("Data was successfully fetched from DB. Params: userId={}", userId);
```

Comment:
let's change method here
```suggestion
        logger.info("Data was successfully fetched from DB. Params: userId={}", userId);
```